pkg/cmd/issue/edit/edit.go

```diff
@@ -113,6 +117,10 @@ func NewCmdEdit(f *cmdutil.Factory, runF func(*EditOptions) error) *cobra.Comman
 				return cmdutil.FlagErrorf("field to edit flag required when not running interactively")
 			}
 
+			if opts.Interactive && len(opts.SelectorArgs) > 1 {
+				return cmdutil.FlagErrorf("multiple issues cannot be edited interactively")
+			}
+
```

Comment:
I wonder if this is the spot we should validate that if multiple issues are being edited that they all belong to the same host and same repo. I feel like doing input validation in `RunE` is a pattern we have used to great effect and it would be better to do the validations here before doing any actual work below in `editRun`.

Reply:
I could look into it, but it would move parsing the repo from `editRun` to here. Since we were already parsing the repo for each issue - since we have to parse each issue for a possible URL anyway - it made sense to alter that function.

Reply:
That is fair. Let's leave it for now and we can come back to it later if anyone feels strongly enough that the validations should occur here instead.